tests/data_test.py

```diff
@@ -30,15 +32,48 @@ def test_datasets(ds_cls: type[VisionDataset]) -> None:
 
 
 def test_audio_dataset() -> None:
-    """Tests basic functionality of an audio dataset base class."""
+    """Tests basic functionality of the base audio dataset class."""
     x = torch.rand(1, 10)
     audio_dir = Path(r"Sample path")
     dataset = PBAudioDataset(x=x, audio_dir=audio_dir)
 
     assert dataset is not None
     assert len(dataset) == len(x)
+
+    
+def test_audio_dataset() -> None:
+    root_dir = Path("~/Data").expanduser()
```

Comment:
this only works on your machine, right?

Reply:
I think this is copied from the other tests which are marked as slow so that they are skipped by the CI, but you can run locally

Reply:
Ah I didn't think it fully through, I thought It just opens the Data dir on the machine, yes copied from other tests, will this be a problem?

Reply:
not a problem, just mark as slow